store/store.go

```diff
@@ -262,7 +262,12 @@ func (s *SnapUbuntuStoreRepository) Snap(name, channel string, auther Authentica
 	case resp.StatusCode == 404:
 		return nil, ErrSnapNotFound
 	case resp.StatusCode != 200:
-		return nil, fmt.Errorf("SnapUbuntuStoreRepository: unexpected HTTP status code %d while looking for snap %q/%q", resp.StatusCode, name, channel)
+		tpl := "Ubuntu CPI service returned unexpected HTTP status code %d while looking for snap %q/%q"
```

Comment:
given that we don't use the notation anymore on the command line  itself maybe time to change this to snap %q in channel %q ?